### DISCUSSION TITLE
toggle hidden files in file picker

### DISCUSSION THREAD

**Discussion by odama626** [helix-editor/helix#7291] [Ideas]
I see that there is a config that can be set to enable hidden files in the file picker but when I need to open a hidden file it is normally a  1 off and I don't always know the exact path, especially if I am looking for build artifacts or workflows in the .github folder,

could we add a shortcut to temporarily toggle the visibility of hidden files while the file picker is open?

**Comment by connortsui20** [comment]
https://github.com/helix-editor/helix/issues/7846
